source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1514,11 +1514,31 @@ servers. For a "node is recovering" error, single-threaded clients MUST NOT
 mark the topology as "stale". If a node is recovering for some time, an
 immediate scan may not gain useful information.
 
-The client SHOULD clear its connection pool for the server.
+The following subset of "not master" errors is defined to be "server shutting
+down" errors:
```

Comment:
According to the tables above, this is a subset of "node is recovering" errors.

Reply:
I see, thank you.